backend/django/create-user-model.mo.md

```diff
@@ -0,0 +1,74 @@
+# Context:
+You have a Django project installed. You need to handle different types of users
+
+# Prerequisite:
+Have the project installed.
+https://github.com/bamlab/dev-standards/blob/master/backend/django/getting-started.mo.md
+
+# Situation:
+You have a basic user model with a name.
+You want to create another type of user, a Mayor that has a City. 
+You do not want to override the classic user model.
+You want to create a Mayor from the admin
+You want the newly created Mayor to be part of the staff (User that can access the admin)
+
+## Add the Mayor model
+
+- In the models.py file, where the User is defined, create a new class that inherits the User:
+```
+class Mayor(User):
+    city = models.ForeignKey('locations.city')
+
+    class Meta:
+        verbose_name = "Mayor"
+```
+ 
```

Comment:
# Steps missing

- run the `makemigrations` command
- run the `migrate` command

# Checks missing

- run the `showmigrations` command

Reply:
trop stylé ca je connaissais pas